src/redux/reducers/dialogsReducer.js

```diff
@@ -44,12 +44,10 @@ const initialState = {
       message: 'priveti4 ot Marmeladi4a',
     },
   ],
-
-  newMessageBody: '',
 };
 
 const dialogsReducer = (state = initialState, action) => {
-  const body = state.newMessageBody;
+  const body = action.newMessageBody;
```

Comment:
Consider make the "action" object the same structure throughout your app - keep the data under "payload" key, not under a custom key.

Reply:
Don't understand what you mean

Reply:
If all your actions will follow the same pattern like "{ type: '...', payload: '...' }", it will help you write less buggy code + typing with TypeScript would be a lot easier.